scipy/stats/_stats_py.py

```diff
@@ -571,6 +571,18 @@ def mode(a, axis=0, nan_policy='propagate'):
     axis : int or None, optional
         Axis along which to operate. Default is 0. If None, compute over
         the whole array `a`.
+
+        .. warning::
+            Unlike other reduction functions (e.g. `skew`, `kurtosis`), the
+            default behavior of `mode` is is to preserve the the axis it acts
+            along. In SciPy 1.11.0, the default behavior will change to become
+            consistent with other reduction functions: consume (remove) the
+            axis it acts along. At that time, an optional parameter
+            ``keepdims`` (like that of `skew`, `kurtosis`) will become
+            available, and users will have the option of setting
+            ``keepdims=True`` to preserve all the original dimensions of the
+            input.
```

Comment:
The use here of single vs double backticks is inconsistent. I always forget which one too, but a mix is probably not working. 😅 

Also, there's another mention of "Support for non-numeric arrays has been deprecated and will be removed in the second release after SciPy 1.9." in the Notes that should be replaced.

Reply:
I'll improve the other "second release after 1.9.0" bit.

Re:backticks:, unfortunately, it is (arguably) correct usage in each case. I'm pretty consistent about applying the rules as I've interpreted them, although that is debatable.

The only guidance in the [standards](https://numpydoc.readthedocs.io/en/latest/format.html) is "Enclose variables in single backticks" and "For inline display use double backticks". 

So I've generalized these rules to:
* Functions, classes, parameters, objects, attributes, methods, and returns that are given names in our existing documentation get single backticks.
* All other text that is intended to render as "code" gets double backticks.

I don't apply these rules consistently to Python literals because I don't think everyone else agrees that, e.g., values that can be passed into functions like `True` an `1` should render as code (but I do). I would agree that eight characters is a bit excessive to type the word `True`.

Here's how I applied these rules:
`skew`, `kurtosis`, and `mode` are functions. (Note that in this case, single backticks will make links to the appropriate function documentation.)
`keepdims=True` is none of the special cases that get single backticks. Double backticks makes it render as code.
`keepdims` is the one that is questionable. It is not a parameter of the function currently, so I chose double backticks. You could argue that since it is a parameter of other functions and is planned to be a parameter of this function in the future, it should be single backticks.

Reply:
I stand corrected, and thanks a lot for the very detailed answer! 🙏
Whenever I absorb this information, I tend to forget it again right away, because my brain refuses to save the significance of single-vs-double quotes of any kind (and where I do remember it took a long time). sorry 😅